webpack.config.js

```diff
@@ -34,6 +34,10 @@ const EXPLORER_APP = {
 
     GOOGLE_ANALYTICS_ID: undefined,
     REACT_APP_SENTRY_DSN: undefined,
+
+    REACT_APP_SUBGRAPH_URL_MAINNET: undefined,
+    REACT_APP_SUBGRAPH_URL_GNOSIS_CHAIN: undefined,
+    REACT_APP_SUBGRAPH_URL_GOERLI: undefined,
```

Comment:
@anxolin this was missing

Without that env vars are not picked up

I don't know exactly why this is required, but the important part is it works now

Reply:
Okay, we should migrate to CRA soonish and continue cleaning up so projects are more homogeneus